addons/SpellBook/SpellBook.lua

```diff
@@ -0,0 +1,382 @@
+_addon.name     = 'SpellBook'
+_addon.author   = 'SigilBaram'
+_addon.version  = '0.0.1'
+_addon.commands  = {'spellbook','spbk'}
```

Comment:
`spbk` feels a bit long for a "short" command. Maybe just `sb`? Would conflict with *Scoreboard*, but from what I hear it's not used much anymore due to *Parse*.